bot.js

```diff
@@ -38,15 +39,19 @@ class Bot {
             return;
         }
 
-        if (config.DEBUG) this.log("Starting ", cmd.join(' '));
-
         this.proc.stderr.pipe(split2()).on('data', (data) => {
             if (this.ignore)  return;
             const errline = data.toString().trim();
             if (errline === "") return;
             this.error(`stderr: ${errline}`);
 
             if (config.ogspv) this.pv.postPvToChat(errline);
+            if (config.aichat) {
+                const chat_match = /(DISCUSSION|MALKOVICH):(.*)/.exec(errline)
```

Comment:
What bots support this function? Also why does it not hook into pv code? `pv` object allows `postPvToChat` to be replaced by a function that could just does what line 50-53 does here.

The code itself does look sound so if you think there is value in supporting both pv and this discussion option then it absolutely has to be done like this. I'm fine either way.

Reply:
To explain the pv code: It should have been an interface that has just `postPvToChat` where you hook up the correct implementation based on the requested bot. However because my limited Js knowledge on how to make interfaces in Js (with Ts i would totally have done that haha), we ended up with 1 pv class that modifies itself based on the requested bot.

Reply:
It's interesting that you think gtp2ogs should be made to support certain bots, rather than someone making a bot to use with gtp2ogs. I've done the latter, and found this handy. Other bots may use it as well.
I am not sending PVs, and PVs are broken at OGS when saving SGFs.

Reply:
yeah i guess that makes sens. All ok then.